monkestation/code/game/objects/items/melee/armaments.dm

```diff
@@ -0,0 +1,146 @@
+/obj/item/armament
```

Comment:
I'd personally split these into files per weapon and just have an armaments folder so its easier to find specific things